swarm/pss/pss_test.go

```diff
@@ -825,6 +824,94 @@ outer:
 
 }
 
+func TestDeduplication(t *testing.T) {
+	var err error
+
+	clients, err := setupNetwork(3)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var addrsize = 32
+	var loaddrhex string
```

Comment:
Nitpick, but this is a good candidate for a for loop from 0 to 3, rather than copy-paste the same code three times. We also have some magic indices within the resulting string that is duplicated.

Reply:
Yeah I want to make condensation of these tests a separate PR with a "first good issue" tag.